react/features/base/media/reducer.js

```diff
@@ -138,3 +149,27 @@ ReducerRegistry.register('features/base/media', combineReducers({
     audio: _audio,
     video: _video
 }));
+
+/**
+ * Stores the last applied transformation to a stream.
+ *
+ * @private
+ * @param {Object} state - The state of the feature.
+ * @param {Object} action - The Redux acrion.
+ * @returns {Object}
+ */
+function _storeVideoTransform(state, action) {
+    const { streamId, transform } = action;
+    let { transforms } = state;
+
+    if (!transforms) {
+        transforms = {};
+    }
+
+    transforms[streamId] = transform;
```

Comment:
Well, that's technically a modification of the redux state. Though you do get state and compute a next state in order to not modify `state`'s props, you're still modifying the very `transforms` object referenced by `state`.